shell/src/main/java/alluxio/cli/fsadmin/report/JobServiceMetricsCommand.java

```diff
@@ -52,6 +59,31 @@ public int run() throws IOException {
       mPrintStream.print(String.format("Status: %-10s", statusSummary.getStatus()));
       mPrintStream.println(String.format("Count: %s", statusSummary.getCount()));
     }
+
+    mPrintStream.println();
+    mPrintStream.println("Last 10 Activities:");
+
+    List<JobInfo> lastActivities = jobServiceSummary.getRecentActivities();
+
+    for (JobInfo lastActivity : lastActivities) {
+      mPrintStream.print(String.format("Timestamp: %-30s",
+          CommonUtils.convertMsToDate(lastActivity.getLastStatusChangeMs(), mDateFormatPattern)));
+      mPrintStream.print(String.format("Job Id: %-20s", lastActivity.getJobId()));
+      mPrintStream.println(String.format("Status: %s", lastActivity.getStatus()));
+    }
+
+    mPrintStream.println();
+    mPrintStream.println("Last 10 Failures:");
```

Comment:
This is not 100% accurate right, its more like `last 10 failures in recent history`?

Reply:
Yeah I should say Last 10 **** in recent history for both.